pyfar/plot/shortcuts/shortcuts.json

```diff
@@ -1,28 +1,35 @@
 {
 "plots": {
     "line.time": {
-        "key": ["1"],
+        "key": ["1", "T"],
+        "key_verbose": ["1", "shift+t"],
         "info": "line.time"},
     "line.freq": {
-        "key": ["2"],
+        "key": ["2", "F"],
+        "key_verbose": ["2", "shift+f"],
         "info": "line.freq"},
     "line.phase": {
-        "key": ["3"],
+        "key": ["3", "P"],
+        "key_verbose": ["3", "shift+p"],
         "info": "line.phase"},
     "line.group_delay": {
-        "key": ["4"],
+        "key": ["4", "G"],
+        "key_verbose": ["4", "shift+g"],
         "info": "line.group_delay"},
     "line.spectrogram": {
-        "key": ["5"],
+        "key": ["5", "S"],
+        "key_verbose": ["5", "shift+s"],
         "info": "line.spectrogram"},
     "line.time_freq": {
         "key": ["6"],
         "info": "line.time_freq"},
```

Comment:
Anyone with a good mnemonic style idea for this one?

Reply:
ctr+shift+t, ctr+shift+f